src/model.ts

```diff
@@ -298,7 +317,10 @@ export class Model extends EventBus<any> implements CommandDispatcher {
         this.state.recordChanges.bind(this.state),
         (command: CoreCommand) => {
           this.isReplayingCommand = true;
-          this.dispatchToHandlers([this.range, ...this.corePlugins], command);
+          this.dispatchToHandlers(
+            [this.range, ...this.corePlugins, ...this.coreViewsPlugins],
```

Comment:
this can be tested:
A core command dispatching an `UPDATE_CELL` command should invalidate evaluation

Reply:
Done: https://github.com/odoo/o-spreadsheet/pull/1817/files#diff-cf5735e699ad89c54032c224139a1cca3f60889cd5d5d30381d59043e3f47a63R201